### PR TITLE
[twig-bundle] uncomment `encore_*_tags()` when 5.4 is used

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | https://github.com/symfony/symfony-docs/issues/15457

Needs https://github.com/symfony/symfony/pull/41205

### CODE REVIEW

File: symfony/twig-bundle/5.4/config/packages/twig.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+twig:
+    default_path: '%kernel.project_dir%/templates'
+    debug: '%kernel.debug%'
+    strict_variables: '%kernel.debug%'
```
Comment (weaverryan) [inline symfony/twig-bundle/5.4/config/packages/twig.yaml:4]: It looks like this file is a bit simpler in the current, 5.0 recipe: https://github.com/symfony/recipes/blob/master/symfony/twig-bundle/5.0/config/packages/twig.yaml

Everything else looks good

Comment (nicolas-grekas) [reply symfony/twig-bundle/5.4/config/packages/twig.yaml]: updated, thanks

**Review by weaverryan — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
PR ready!

**Review by javiereguiluz — APPROVED** [review]
Nice one! Thanks.

**Review by chalasr — APPROVED** [review]

**Comment by stof** [comment]
@nicolas-grekas something looks wrong in the comment of the github-actions-bot. There is no diff between 5.3 and 5.4, while it is the relevant one here as you are adding the 5.4 one.

File: symfony/twig-bundle/5.4/templates/base.html.twig [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>{% block title %}Welcome!{% endblock %}</title>
+        {# Run `composer require symfony/webpack-encore-bundle` to start using Symfony UX #}
```
Comment (dunglas) [inline symfony/twig-bundle/5.4/templates/base.html.twig:6]: Shouldn't this comment be removed?

Comment (nicolas-grekas) [reply symfony/twig-bundle/5.4/templates/base.html.twig:6]: we can keep it, it's a nice ad ;)

**Pull request merged**